damnit/backend/extract_data.py

```diff
@@ -106,6 +108,24 @@ def extract_in_subprocess(
     for m in match:
         args.extend(['--match', m])
 
+    if sandbox:
+        bubblewrap = Bubblewrap()
+        with contextlib.suppress(Exception):
+            bubblewrap.add_bind_proposal(proposal)
+        bubblewrap.add_bind_venv(Path(sys.executable))
+        if python_exe and Path(sys.executable) != Path(python_exe):
+            bubblewrap.add_bind_venv(Path(python_exe))
+        bubblewrap.add_bind(Path(__file__).parents[1] / 'ctxsupport') # ctxsupport_dir
```

Comment:
Depends on how it's installed, if DAMNIT was `pip install`'ed normally then yes it would be under the python purelib, but if it was an editable `pip install -e`' then it could be anywhere outside of the python paths. Adding it explicitly here makes it possible to test/run when in a development environment.

I guess it could be made conditional and only added in if it's not already covered by another previous bind mount, but the cost of these mounts is tiny so there's not much point to avoiding them. Or sandboxing could be disabled when an editable install is detected :thinking:

Reply:
Ah ok. Yeah we should definitely support editable installs, that's how it's deployed on Maxwell :stuck_out_tongue: